codereview.java

```diff
@@ -0,0 +1,36 @@
+public class RuntimeIssuesDemo {
+
+    public static void main(String[] args) {
+        int[] numbers = {1, 2, 3};
+        System.out.println(numbers[3]); 
```

Comment:
<b>code: System.out.println(numbers[3]); - </b>
ArrayIndexOutOfBoundsException might occur due to accessing out of bounds index